entrypoint.sh

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env bash
+set -Eeo pipefail
+# TODO swap to -Eeuo pipefail above (after handling all potentially-unset variables)
+
+# usage: file_env VAR [DEFAULT]
+#    ie: file_env 'XYZ_DB_PASSWORD' 'example'
+# (will allow for "$XYZ_DB_PASSWORD_FILE" to fill in the value of
+#  "$XYZ_DB_PASSWORD" from a file, especially for Docker's secrets feature)
+file_env() {
+	local var="$1"
+	local fileVar="${var}_FILE"
+	local def="${2:-}"
```

Comment:
This file was not [written by me.](https://github.com/docker-library/postgres/blob/master/13/buster/docker-entrypoint.sh) I just re-used it because I wanted to set the Postgres process in the [background](https://github.com/medic/cht-postgres/blob/arch-v3/tests/entrypoint.sh#L322) and be able to run tests against it. It already exists in the main image just that the Postgres process is the main process.